src/support/azext_support/custom.py

```diff
@@ -272,3 +280,102 @@ def pre_operations(self):
         super().pre_operations()
         args = self.ctx.args
         _check_name_availability_no_subscription(self.cli_ctx, str(args.communication_name), "Microsoft.Support/communications")
+
+class FileWorkspaceCreateNoSubscription(_CreateNoSubscriptionFileWorkspace):
+    def pre_operations(self):
+        from azext_support._validators import _check_name_availability_no_subscription
+        super().pre_operations()
+        args = self.ctx.args
+        file_workspace_name = str(args.file_workspace_name)
```

Comment:
Pass str directly in the function call

Reply:
Keeping in both instance because it makes the code easier to read